doc/devel/development_setup.rst

```diff
@@ -34,24 +34,31 @@ repository, you should first fork this repository by *clicking* the
 This creates a copy of the code under your account on the GitHub server. See `the GitHub
 documentation <https://docs.github.com/get-started/quickstart/fork-a-repo>`__ for more details.
 
-Decide whether to work locally or in GitHub Codespaces
-======================================================
+Set up development environment
+==============================
 
 You can either work locally on your machine, or online in
 `GitHub Codespaces <github-codespaces_>`_, a cloud-based in-browser development
-environment.  If you are making a one-off, relatively simple change then
-working in GitHub Codespaces can be a good option because most of the setting
-up is done for you and you can skip the next few sections!  If you are making
-extensive or frequent contributions to Matplotlib then it is probably worth
-taking the time to set up on your local machine: As well as having the
-convenience of your local familiar tools, you will not need to worry about
-Codespace's monthly usage limits.
+environment.
+
+
+:local: If you are making extensive or frequent contributions to Matplotlib then it
+        is probably worth taking the time to set up on your local machine: As well as
+        having the convenience of your local familiar tools, you will not need to worry
+        about Codespace's monthly usage limits.
+
+:codespaces: If you are making a one-off, relatively simple, change then working in
+            GitHub Codespaces can be a good option because most of the setting
+            up is done for you and you can skip the next few sections!
```

Comment:
```suggestion
            up is done for you and you can skip the next few sections.
```
This is a simple statement, no need to put emphasis on it using an exclamation mark.